### PR TITLE
Added Russian transation for the pop-up comment of the shortcut

### PR DESCRIPTION
**Pull request by TotalCaesar659**
This change is [Reviewable](https://reviewable.io/reviews/grayhatter/utox/257)

### CODE REVIEW

File: src/utox.desktop [hunk 1]
```diff
@@ -3,6 +3,7 @@ Version=1.0
 Type=Application
 Name=uTox
 Comment=A lightweight Tox client
+Comment[ru]=Нетребовательный к ресурсам клиент Tox
```
Comment (cebe) [inline src/utox.desktop]: I do not think `lightweight` only refers to computer resources. I am not a native speaker but something like `легкий`, `простой`, or `несложный` would be more fitting imo.

**Comment by TotalCaesar659** [comment]
OK, I just tried to satisfy it. Though, you can correct it to "Легкий клиент Tox" if you want.

**Comment by GrayHatter** [comment]
@cebe thoughts on this one?

**Comment by cebe** [comment]
@GrayHatter change it as `Легкий клиент Tox` and close this one?

**Comment by alexeysvrv** [comment]
So, I support Russian their native language, I suggest using a combination of a `легковесный клиент` It is more accurately suited and positioned uTox

**Pull request merged**